articles/quickstart/native/android/interactive.md

```diff
@@ -0,0 +1,175 @@
+
+Add the [Auth0 Android](https://github.com/auth0/Auth0.Android) SDK into your project. The library will make requests to the Auth0's Authentication and Management APIs.
+
+In your app's `build.gradle` dependencies section, add the following:
+
+```groovy
+implementation 'com.auth0.android:auth0:2.+'
+```
+
+Ensure you target Java 8+ byte code for Android and Kotlin plugins respectively.
+
+## Add manifest placeholders {{{ data-action=code data-code="build.gradle#10:12" }}}
+
+Add manifest placeholders required by the SDK. The placeholders are used internally to define an `intent-filter` that captures the authentication callback URL. For this, the Auth0 tenant domain and the scheme that take part in the callback URL must be set.
+
+You do not need to declare a specific `intent-filter` for your activity, because you have defined the manifest placeholders with your Auth0 **Domain** and **Scheme** values and the library will handle the redirection for you.
+
+::: note
+We've used a value of `demo` for `auth0Scheme` here, so that a custom URL scheme can be used for the URL that Auth0 redirects to after login. The alternative is `https` if you want to use [Android App Links](https://auth0.com/docs/applications/enable-android-app-links). You can read more about setting this value in the [Auth0.Android SDK README](https://github.com/auth0/Auth0.Android#a-note-about-app-deep-linking).
+:::
+
+## Configure your application {{{ data-action=code data-code="strings.xml#2:3" }}}
+
+For the SDK to function properly, you must set the following properties in `strings.xml`:
+
+- `com_auth0_domain`: The domain of your Auth0 tenant. Generally, you can find this in the Auth0 Dashboard under your Application's Settings in the Domain field. If you are using a [custom domain](https://auth0.com/docs/custom-domains), you should set this to the value of your custom domain instead.
+- `com_auth0_client_id`: The ID of the Auth0 Application you set up earlier in this quickstart. You can find this in the Auth0 Dashboard under your Application's Settings in the Client ID field.
+
+Ensure that the `android.permissions.INTERNET` permission is specified in the `AndroidManifest.xml` file:
+
+```xml
+<uses-permission android:name="android.permission.INTERNET" />
+```
+
+Run **Sync Project with Gradle Files** inside Android Studio or execute `./gradlew clean assembleDebug` from the command line.
+
+::: note
+For more information about using Gradle, check the [Gradle official documentation](https://gradle.org/getting-started-android-build/).
+:::
+
+## Add login to your app {{{ data-action=code data-code="MainActivity.kt#6:38" }}}
+
+[Universal Login](/hosted-pages/login) is the easiest way to set up authentication in your application. We recommend using it for the best experience, best security and the fullest array of features.
+
+In the `onCreate` method, create a new instance of the `Auth0` class to hold user credentials:
```

Comment:
Should the next few paragraphs be in list format?

Reply:
I meant explicitly the end punctuation on line 93.

`In the `onCreate` method, create a new instance of the `Auth0` class to hold user credentials:` 
Should that be a period instead of `:` ? It did not read correctly to me as a list but I thought it might be missing something or just a typo.